css/index.css

```diff
@@ -1,3 +1,294 @@
 /* Use your own code or past solution for Great Idea Web Page CSS here! */
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
 
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+    margin: 0;
+    padding: 0;
+    border: 0;
+    font-size: 100%;
+    font: inherit;
+    vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+    display: block;
+}
+body {
+    line-height: 1;
+}
+ol, ul {
+    list-style: none;
+}
+blockquote, q {
+    quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+    content: '';
+    content: none;
+}
+table {
+    border-collapse: collapse;
+    border-spacing: 0;
+}
 
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', sans-serif;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Your code starts here! */
+
+
+.wrapper {
+    max-width: 900px;
+    width: 100%;
+    margin-left: auto;
+    margin-right: auto;
+}
+nav, header, section {
+    display: flex;
+    padding: 34px 0px 0px;
+}
+nav a, header a, section a {
+    text-decoration: none;
+}
+nav {
+    justify-content: space-between;
+    
+}
+.nav {
+    display: flex;
+    
+}
+nav a {
+    color: #bebebe;
+    padding: 0px 30px;
+    align-self: center;
+    font-weight: bold;
+}
+nav a:first-child {
+    padding-left: 0px;
+}
+
+header a, section a {
+    color: black;
+}
+/*nav a:first-child {
+    margin-left: -33px;
+}
+nav img {
+    margin-left: 88px;
+    margin-top: -10px;
+}*/
+header {
+    border-bottom: 1.5px solid black;
+    padding-bottom: 70px;
+}
+
+header a {
+    border: 1px solid black;
+    margin: 0px 127px 0px;
+    text-align: center;
+    display: flex;
+    font-weight: bold;
+    flex-flow: column;
+    padding: 8px;
+}
+h1 {
+    font-size: 76px;
+    text-align: center;
+    margin-top: 128px;
+    width: 302px;
+    margin-left: 65px;
+    padding: 0px 17px;
```

Comment:
This padding is actually offsetting your `h1` tag, preventing the text from being center. If you remove it, the text in your `h1` tag is centered.